styles/utilities.scss

```diff
@@ -0,0 +1,29 @@
+.hover {
+  span {
```

Comment:
Not sure if this name (`.hover`) is fine, please check alternatives

Reply:
the hover effect is working from ListHeader.module.scss. i dont know how reutilize classes in utilities.scss cant be import in the component

Reply:
![Screenshot from 2022-09-06 09-21-47](https://user-images.githubusercontent.com/10248108/188659945-83dd2127-2051-4498-a883-913521843940.png)

Reply:
![Screenshot from 2022-09-06 09-22-22](https://user-images.githubusercontent.com/10248108/188660089-a2f714d6-3c41-48da-b44c-f9ee6731cdc8.png)

Reply:
@Eghinner please help here

Reply:
@import utilities in global.scss